poetry/console/commands/add.py

```diff
@@ -40,11 +40,18 @@ class AddCommand(EnvCommand, InitCommand):
             "Output the operations but do not execute anything (implicitly enables --verbose).",
         ),
     ]
-
-    help = """The add command adds required packages to your <comment>pyproject.toml</> and installs them.
-
-If you do not specify a version constraint, poetry will choose a suitable one based on the available package versions.
-"""
+    help = (
+        "The add command adds required packages to your <comment>pyproject.toml</> and installs them.\n\n"
+        "If you do not specify a version constraint, poetry will choose a suitable one based on the available package versions.\n\n"
+        "You can specify a package in the following forms:\n"
+        "  - A single name (<b>requests</b>)\n"
+        "  - A name and a constraint (<b>requests ^2.23.0</b>)\n"
```

Comment:
Hello @dgnsrekt,

thanks a lot for your contribution. :+1: 

I think for the variant name + constraint the example should be `requests@^2.23.0`, shouldn't it?

fin swimmer

Reply:
That maybe true. I'm not quite sure. But, I copied it from the init command sections. So, if its wrong it may need to be fixed. I'll look into it and create a pull request if its incorrect.

Reply:
#2574 fixes the issue in the add and init command documentation.